test/functional/feature_rbf.py

```diff
@@ -618,6 +618,15 @@ def test_no_inherited_signaling(self):
         assert_equal(True, self.nodes[0].getmempoolentry(optin_parent_tx['txid'])['bip125-replaceable'])
         assert_raises_rpc_error(-26, 'txn-mempool-conflict', self.nodes[0].sendrawtransaction, replacement_child_tx["hex"], 0)
 
+        self.log.info('Check that the child tx can still be replaced (via a tx that also replaces the parent)')
+        replacement_parent_tx = self.wallet.send_self_transfer(
+            from_node=self.nodes[0],
+            utxo_to_spend=confirmed_utxo,
+            sequence=0xffffffff,
+            fee_rate=Decimal('0.03'),
+        )
+        self.wallet.get_utxo(txid=optout_child_tx['txid'])  # Update wallet utxo state
```

Comment:
nit: could be useful to add an assertion that the child tx is no longer in the mempool?